WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -610,6 +613,20 @@ - (void)refreshPostHeaderView
             [self.postHeaderView setAvatarImage:image];
         }];
     }
+    
+    FollowCommentsService *service = [[FollowCommentsService alloc] initWithPost:self.post];
+    self.postHeaderView.showsFollowConversationButton = [service canFollowConversation];
+}
+
+- (void)refreshSubscriptionStatus
+{
+    __weak __typeof(self) weakSelf = self;
+    FollowCommentsService *service = [[FollowCommentsService alloc] initWithPost:self.post];
```

Comment:
Two things about this:

* We're instantiating `FollowCommentsService` inline in two different places. Wouldn't be better to make it a class property?
* Correct me if I'm wrong, but we will always call the service even though the button is unavailable (hidden), right? Shall we prevent that?

Reply:
> We're instantiating FollowCommentsService inline in two different places. Wouldn't be better to make it a class property?

Good point. Thanks for the suggestion 👍 

> we will always call the service even though the button is unavailable (hidden), right? Shall we prevent that?

Very, very good point. I totally missed this. Thank you for catching this 🙏

Reply:
Fixed! 
045a214
9a0d9f7